save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -242,15 +241,17 @@ class DownloadProjectController(
                         log.error("Error to update test suite with url=$testSuiteUrl, path=${testSuiteRepoInfo.testSuitePaths}")
                     }
             }
+                .flatMap { Flux.fromIterable(it) }
```

Comment:
Will there be any difference between `flatMap { Flux.fromIterable(it) }` and `flatMapIterable { it }`?

Reply:
I don't see any difference, will change to `flatMapIterable` , I guess it's more optimized for iterable